web/src/components/Hero.js

```diff
@@ -117,17 +119,30 @@ query getTasks {
           <div>
             <h1
               className={`
-                text-4xl font-extrabold text-neutral-700
-                lg:text-5xl lg:leading-tight
+                flex flex-col
+                text-[3.5rem] sm:text-[6rem] font-bold text-neutral-700
+                lg:text-[5rem] lg:leading-tight
               `}
             >
-              Develop full-stack web apps{' '}
-              <span className="underline decoration-yellow-500">faster</span>.
+              <div>
+                <span className="-ml-[0.22rem]">Fullstack</span>
+              </div>
+              <div>
+                <span
+                  className={classNames(
+                    'font-jersey text-[4.5rem] sm:text-[8rem] lg:text-[6.35rem] font-normal text-yellow-500 '
+                  )}
+                >
+                  Everything
+                </span>
+                <span>.</span>
+              </div>
             </h1>
 
-            <p className="mt-4 text-xl text-neutral-500 sm:mt-5 lg:text-xl">
-              Rails-like framework for React, Node.js and Prisma. Build your app
-              in a day and deploy it with a single CLI command.
+            <p className="mt-8 text-xl text-neutral-500 sm:mt-5 lg:text-xl lg:leading-relaxed">
+              The Laravel alternative for JavaScript.
```

Comment:
I am starting to think that we should mention Laravel and RoR, but that it shouldn't be the way we define ourselves. I would say, first you read what Wasp is, then you wonder "ok but what is that like in practice", and then we say "Imagine RoR or Laravel's DX, but reimagined from scratch for JS and beyond".